src/testUtils.ts

```diff
@@ -63,6 +65,22 @@ export async function expectVMError(error: string, script: Promise<any>) {
   expect(isError).toBeTruthy()
 }
 
+export async function expectOutOfGas(script: Promise<any>) {
```

Comment:
The functions looks exactly the same as `expectVMError`, it could easily be merged with only adjusted regex `new RegExp(error)` without any hard-coded prefix